scripts/sanity_chk/expr_parser.py

```diff
@@ -10,6 +10,11 @@
 import threading
 import re
 
+ZEPHYR_BASE = os.environ.get("ZEPHYR_BASE")
+sys.path.insert(0, os.path.join(ZEPHYR_BASE, "scripts/dts"))
```

Comment:
IIRC for portability this should be `os.path.join(ZEPHYR_BASE, "scripts", "dts")`  but there's probably already a lot of code that won't work for `os.path.sep` not equal to `'/'`.  Certainly there's material in `sanitycheck` that should be using `os.path.split` if that's a concern.